settings/DevHome.Settings/Views/ExperimentalFeaturesPage.xaml

```diff
@@ -28,13 +28,35 @@
                                 Description="{x:Bind Description, Mode=OneWay}"
                                 Margin="{ThemeResource SettingsCardMargin}"
                                 Visibility="{x:Bind IsVisible, Mode=OneWay}">
-                                <ToggleSwitch IsOn="{x:Bind IsEnabled, Mode=OneWay}" Grid.Column="1">
-                                    <i:Interaction.Behaviors>
-                                        <ic:EventTriggerBehavior EventName="Toggled">
-                                            <ic:InvokeCommandAction Command="{x:Bind ToggledCommand}" />
-                                        </ic:EventTriggerBehavior>
-                                    </i:Interaction.Behaviors>
-                                </ToggleSwitch>
+                                <StackPanel
+                                    Orientation="Horizontal"
+                                    Spacing="16"
+                                    Grid.Column="1">
+                                    <ToggleSwitch IsOn="{x:Bind IsEnabled, Mode=OneWay}">

```

Comment:
Wondering if we should use the `ActionIcon` property in `SettingsCard` instead. That ensures we inherit the styles from the SettingsCard control (forward compatible). But that will make the entire button clickable.
```xml
<ctControls:SettingsCard
    ...
    ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
    IsClickEnabled="True">
```

Related links:
- https://github.com/microsoft/WinUI-Gallery/blob/cb323e1a5dff53e482ce577b48ca311cfe8d00cd/WinUIGallery/SettingsPage.xaml#L120-L128